library/std/src/process.rs

```diff
@@ -1417,6 +1417,13 @@ impl From<fs::File> for Stdio {
 ///
 /// [`status`]: Command::status
 /// [`wait`]: Child::wait
+//
+// We speak slightly loosely (here and in various other places in the stdlib docs) about `exit`
+// rather than `_exit`.  Strictly speaking, an exit status is a value passed to the `_exit` system
+// call, around which `exit` is a library wrapper.  But anyone who knows the difference will not be
+// misled; anyone who does not, does not want to be distracted by some subtle point of UNIX APIs.
+// (Also, the distinction between syscalls and library functions is not standardised, although Unix
+// people typically have common notions.)
```

Comment:
This comes across to me very much like "UNIX people understand this and other people don't need to" elitism. I'm not suggesting that that's the *intent*, but I feel like it's the net effect.

I also don't think it's necessary for the purposes of conveying this point. As far as I can tell, `libc::exit` and `libc::_exit` and the Linux syscall `exit` *all* accept an exit status value, and pass it along. So the point worth conveying here seems more like this:

```suggestion
// When this documentation refers to a status passed to `exit` or similar, this
// loosely refers to any mechanism that exits a process and accepts an exit
// code, including system calls, library calls that make such system calls, or
// returning from a program's main function to a library that invokes such
// system calls.
```

(modulo word-wrapping)

Reply:
I think you have missed something:  This isn't a doc comment.  I agree that this kind of language would be inappropriate in public docs.

The audience is other people who are editing this code or its documentation.  The purpose is to explain to any passing Unix pedants why the docs are the way they are.

Reply:
I didn't think it was a doc comment. It's still documentation, just with an audience of people who read the code. We're friendly to each other, too, not just to users.

That said, I do understand the desire to quell pedantry, UNIXy or otherwise. But I think if we want to avoid attracting or encouraging that kind of UNIX pedantry, it'll also help to avoid playing into divisions based on who has which bits of UNIX ingroup knowledge.